docs/modders/Mod_File_Format.md

```diff
@@ -0,0 +1,140 @@
+< [Documentation](../Readme.md) / [Modding](Readme.md) / Mod File Format
+
+## Fields with description of mod
+
+``` javascript
+ 
+{
+    // Name of your mod. While it does not have hard length limit
+    // it should not be longer than ~30 symbols to fit into allowed space
+    "name" : "My test mod",
+
+    // More lengthy description of mod. No hard limit. This text will be visible in launcher.
+    // This field can use small subset of HTML, see link at the bottom of this page.
+    "description" : "My test mod that add a lot of useless stuff into the game",
+
+    // Author of mod. Can be nickname, real name or name of team
+    "author" : "Anonymous",
+
+    // Full name of license used by mod. Should be set only if you're author of mod
+    // or received permission to use such license from original author
+    "licenseName" : "Creative Commons Attribution-ShareAlike",
+
+    // URL which user can use to see license terms and text
+    "licenseURL" : "https://creativecommons.org/licenses/by-sa/4.0/",
+
+    
```

Comment:
```suggestion
    // approximate size of mod, compressed by zip algorithm, in Mb
    "size" : 25,
```

Reply:
Do we support fractional part, e.g. `"size" : 25.3`?

Reply:
yes